utils/vim/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vim
-PKG_VERSION:=8.0.586
-PKG_RELEASE:=2
-VIMVER:=80
+PKG_VERSION:=8.1
+PKG_RELEASE:=1
+VIMVER:=81
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=ftp://ftp.vim.org/pub/vim/unix/
```

Comment:
Please switch to http and add at least two mirrors such as (in front of main site)
http://www6.uniovi.es/pub/vim/unix/
http://ftp.kaist.ac.kr/vim/unix/
http://ftp2.jp.vim.org/pub/vim/unix/

Source: https://www.vim.org/mirrors.php

Reply:
Okay, now everything seems to be done